src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,345 @@
+        debugger [shape="tab" style="filled" fontcolor="white" fillcolor="#225ea8" label="Narratives debugger\nto test narrative\nas we go"]
+        
+        auspice -> md [label="copy\nURL" fontcolor="#7fcdbb" fillcolor="#7fcdbb" color="#7fcdbb"]
+        md -> auspice [label="repeat" fontcolor="#7fcdbb" fillcolor="#7fcdbb" color="#7fcdbb" splines=curved]
+        md -> debugger [label="drag &\ndrop" fontcolor="#1d91c0" fillcolor="#1d91c0" color="#1d91c0"]
+        debugger -> md [label="repeat" fontcolor="#1d91c0" fillcolor="#1d91c0" color="#1d91c0" splines=curved]
+    }
+
+In the future we plan to add more and more editing capability into the debugger, but for now any changes to the narrative must be made in the markdown file itself.
+When you are happy with the end result, :ref:`see below for how to publish it on nextstrain.org <sharing>`.
+
+
+
+Understanding the structure of narrative slides
+===================================================
+
+
+Title slide (frontmatter)
+-------------------------
+
+The opening (title) slide is defined in the markdown file by an opening `YAML <https://learnxinyminutes.com/docs/yaml/>`__ frontmatter block, which is the part between the two ``---`` lines in the above example.
+This defines a number of key-value pairs which we transform into the slide you see, all of which are optional except ``title`` and ``dataset``.
+The possible content which can be rendered is listed below, in the order they would appear on screen:
+
+#. The main title is taken from the ``title`` key.
+#. The authors are then listed; these can be provided via ``authors`` and ``authorsLinks`` which should either both be strings or both be arrays of the same length. The ``authorsLinks`` is optional but recommended!
+#. Any translators are then listed, encoded in the same format as the authors but using keys ``translators`` and ``translatorLinks``.
+#. The abstract, defined by ``abstract`` is a string which will be rendered as markdown [#f1]_.
+#. When the narrative was first created (``date``) and when it was most recently updated (``updated``) is then displayed.
+#. Finally, any applicable license is shown, as defined by ``license`` and ``licenseLink``.
+#. The ``dataset`` is required and defines the data view in the right hand side of screen (see :ref:`see below <linking-view-to-url>`).
+
+
+Normal slides
+-------------
+
+The rest of the markdown file defines one or more slides, where each slide is defined by a level 1 heading which is also a link to a dataset on nextstrain.org [#f2]_ and a section of markdown which represents the slide's content:
+
+.. graphviz::
+    :align: center
+
+    strict digraph {
+        title [shape="box" style="filled" fillcolor="#feb24c" label="# [Slide Title](nextstrain.org dataset URL)"]
+        body [shape="box" style="filled" fillcolor="#fd8d3c" label="Markdown content of slide"]
+        title -> body [style="invis"]
+    }
```

Comment:
It's kinda weird to me to use a graphviz diagram to represent the Markdown contents of a single slide. My first impression upon reading was that each node (`title` and `body`) were representing different _slides_ instead of parts of a single slide. Why not use a `.. code-block:: markdown` block like later?

Reply:
Ha! Yeah I got carried away with my new graphviz expertise

Reply:
Done - https://github.com/nextstrain/docs.nextstrain.org/compare/03a1276b5dfebbb8d81a3447d64cd31239b9f4e4..477e27d73e3cde37e40c662e0b3aea0a9c4cb987